languages/javascript/exercises/concept/strings/.docs/introduction.md

```diff
@@ -1,5 +1,26 @@
-Strings are useful for holding data that can be represented in text form. Some
-of the most-used operations on strings are to check their `length`, to build
-and concatenate them using the `+` and `+=` string operators, checking for the
-existence or location of substrings with the `indexOf()` method, or extracting
-substrings with the `substring()` method.
+Strings are useful for holding data that can be represented in text form.
+There are two ways to access an individual character in a string.
+
+The first is the charAt() method:
+
+```javascript
+'cat'.charAt(1)
+// => "a"
+```
+
+The other way is to treat a string as an array-like object, where individual characters correspond to a numerical index(starts at zero):
```

Comment:
```suggestion
The other way is to treat a `string` as a list of characters, where individual characters correspond to a numerical index (starts at zero):
```

Reply:
I''m okay lying to the student here. It's not individual characters, but code points.